TODO

```diff
@@ -1394,8 +1394,6 @@ Features:
     - kmod-libs (only when called from PID 1)
     - libblkid (only in RootImage= handling in PID 1, but not elsewhere)
     - libpam (only when called from PID 1)
-    - bzip2, xz, lz4 (always — gzip and zstd should probably stay static deps the way they are,
-      since they are so basic and our defaults)
```

Comment:
hmm, so the bzip2 dep is untouched by your PR afaics. Any chance you can cover that too? it's like the one most people probably want to get rid off in real systems too.

at the very least the TODO entry should be left in place, just with the modules you covered dropped.